.travis.yml

```diff
@@ -60,3 +57,7 @@ cache:
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
+  - git fetch --all && git config --global user.email "-" && git config --global user.name "-"
```

Comment:
@eugenepeh may i get the purpose of `git config --global user.email "-" && git config --global user.name "-"` in the run command?

Reply:
There are some git commands that cannot be performed without the user.name and user.email value. We simply use the `-` value to bypass that need.

Reply:
ahh ok, noted.